src/tools/winutil.hpp

```diff
@@ -62,6 +66,44 @@ inline int os_specific_y_offset() {
 #endif
 }
 
+// The default scale should be the largest that the user's screen can fit all three
+// BoE application windows (because they should probably default to match each other).
+inline float fallback_scale() {
+	static float _fallback_scale = 0;
```

Comment:
1. I don't like variable names beginning with an underscore. Wouldn't just `scale` do just fine?
2. This is a pretty big function, does it really need to be defined inline in the header?

Reply:
1. Yes
2. Probably not. I just need to find a different header/source file that is included in the Common sources where it would make sense to be forward-declared/defined. Because if I declare it in winutil.hpp, the definition would have to be repeated in the 3 different winutil.*.cpp/mm files.